packages/@aws-cdk/aws-stepfunctions-tasks/lib/emrcontainers/base-types.ts

```diff
@@ -0,0 +1,270 @@
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+
+/**
+ * The information about job driver for Spark submit.
+ */
+export interface SparkSubmitJobDriver {
+
+  /**
+   * The entry point of job application.
+   * Length Constraints: Minimum length of 1. Maximum length of 256.
```

Comment:
with the constraints that you document here, please add validation if the value provided is not a JSONPath